src/pages/ProfileSettings.tsx

```diff
@@ -1,20 +1,261 @@
+                            d="M9.37333 6L10 6.62667L3.94667 12.6667H3.33333V12.0533L9.37333 6ZM11.7733 2C11.6067 2 11.4333 2.06667 11.3067 2.19333L10.0867 3.41333L12.5867 5.91333L13.8067 4.69333C14.0667 4.43333 14.0667 4 13.8067 3.75333L12.2467 2.19333C12.1133 2.06 11.9467 2 11.7733 2ZM9.37333 4.12667L2 11.5V14H4.5L11.8733 6.62667L9.37333 4.12667Z"
+                            fill="#77809C"
+                          />
+                        </svg>
+
+                        <div className="text-xs font-normal">Edit</div>
+                      </Button>
+                    </AlertDialogTrigger>
+                    <AlertDialogContent className="m-0 p-0 ">
+                      <AlertDialogHeader className="px-6 py-2 border-b">
+                        <AlertDialogTitle className="text-left flex flex-row justify-between items-center">
+                          Edit Billing Details
+                          <AlertDialogCancel className="bg-transparent border-none">
+                            <X />
+                          </AlertDialogCancel>
+                        </AlertDialogTitle>
+                      </AlertDialogHeader>
+                      <div className="flex-1 gap-4 justify-center text-left flex-col lg:gap-4 w-full px-6">
+                        <InputWithLabel label="Address" type="text" placeholder="Type here ..." classNameDiv="mt-2" />
+                        <div className="flex flex-col lg:grid-cols-2 lg:grid lg:gap-x-4">
+                          <InputWithLabel
+                            label="Locality"
+                            type="text"
+                            placeholder="Type here ..."
+                            classNameDiv="mt-2"
+                          />
+                          <InputWithLabel
+                            label="Landmark"
+                            type="text"
+                            placeholder="Type here ..."
+                            classNameDiv="mt-2"
+                          />
+                          <InputWithLabel
+                            label="Pin code"
+                            type="text"
+                            placeholder="Type here ..."
+                            classNameDiv="mt-2"
+                          />
+                          <InputWithLabel label="City" type="text" placeholder="Type here ..." classNameDiv="mt-2" />
+                          <InputWithLabel label="State" type="text" placeholder="Type here ..." classNameDiv="mt-2" />
+                          <InputWithLabel label="Country" type="text" placeholder="Type here ..." classNameDiv="mt-2" />
+                        </div>
+                      </div>
+
+                      <AlertDialogFooter className="px-6 py-2 flex flex-row items-center border-t mt-6 gap-4 justify-end">
+                        <AlertDialogCancel className="mb-2 md:mb-0 border-blue-400 text-blue-400 text-xs font-normal font-poppins">
+                          Cancel
```

Comment:
We just need two buttons here for cancel and save, no need to use AlertDialog Components